addons/product/views/product_tag_views.xml

```diff
@@ -25,9 +27,11 @@
         <field name="name">product.tag.tree</field>
         <field name="model">product.tag</field>
         <field name="arch" type="xml">
-            <tree string="Product Tags">
+            <tree string="Product Tags" multi_edit="1">
                 <field name="name"/>
-                <field name="color" widget="color_picker"/>
+                <field name="product_template_ids" widget="many2many_tags" optional="show"/>
+                <field name="product_product_ids" widget="many2many_tags" string="Product Variants" optional="show"/>
+                <field name="color" widget="color" readonly="1" optional="hide"/>
```

Comment:
Why hide the color? IMO this field is the most interesting one to be shown in that view.

Reply:
according to spec, it is like that :)